src/components/WelcomeBox.css

```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
+ :root[platform="mac"] .welcomebox .shortcutKey,
+ .launchpad-root[platform="mac"] .welcomebox .shortcutKey {
+   --monospace-font-family: monospace;
```

Comment:
So I'm not very fond of the monospace font here. It looks like with `font-family: monospace` on a mac, Firefox may end up using Courier, which is not something we have elsewhere in DevTools. The `--monospace-font-family` CSS variable is actually defined so that we can specify a specific monospace font on each platform, so we shouldn't override it, even locally in some branch of the DOM.

It would be more interesting here IMO to try to use San Francisco, which seems to be the font used in the issue's screenshots. [According to Can I Use](https://caniuse.com/#search=system-ui), Firefox for Mac supports the `-apple-system` on macOS. For Chrome, it seems that the `system-ui` value is supported. So maybe something like:

```css
[platform="mac"] .welcomebox .shortcutKey {
  font-family: system-ui, -apple-system, sans-serif;
  font-weight: 600;
}
```

Reply:
This solutions seemed fine both on Chrome and FF. I also changed font-weight to 500, as it looks better.